dependency_metrics/package_managers/yarn.py

```diff
@@ -50,16 +89,14 @@ def parse_yarn_list():
 class Yarn:
     """
     Class to house raw yarn commands
-    NOTE: avoid using --json option because the output includes a lot of
-    irrelevant lines
```

Comment:
Removed this because A, I used `--json` below and didn't want to be a hypocrite. And B, this isn't worthy of a comment. If there is a compelling reason to use `--json` with `yarn list`, we should do so.